<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttackReadyInfo</class>
 <widget class="QWidget" name="AttackReadyInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_blueArmName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>我方武器名字</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bluePeoName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>我方角色名字</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_redPeoName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>敌方人物名字</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_redArmName">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>敌方武器名字</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Hp">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>HP</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_redHpValue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_blueHpValue">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_weiLi">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>175</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>威力</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_mingZhong">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>命中</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_biSha">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>325</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>必杀</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_redWeiLiValue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>175</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_redMingZhongValue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_redBiShaValue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>325</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_blueWeiLiValue">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>175</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_blueMingZhongValue">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_blueBiShaValue">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>325</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lb_startZhandou">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始战斗</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
